### ISSUE TITLE
Investment Board: Develop and field a Net Promoter Score survey for all teams currently using our service

### ISSUE THREAD

**Issue by scayzer** [bcgov/cloud-pathfinder#886]
**Describe the issue**
A Net Promoter Score in a nutshell is how many of your customers would recommend your service to another.

Our business case is nearly complete, and it requires the OCIO Digital Annex which has a Key Results section:
•	How will you know if the investment is successful? Provide quantitative indicators (i.e. KPI) that would allow you to track progress (e.g., project schedule, user satisfaction, cost per transaction, reduced expense, etc.). 
•	Describe your data collection strategy to measure KPIs.
•	Provide targets for progress against the indicators, specifying the reporting frequency and phase/timing (e.g., 30% improvement in user satisfaction within 1 year).
•	Provide insight into any results indicators measured to date (e.g., If you are seeking approval for a beta or live phase).

So far, we've proposed the following KPIs:
• Number of teams onboarded to compliant public cloud platforms (count of teams onboard, target = 4 teams added per quarter in year 1, 6 teams per quarter in year 2, 8 teams per quarter in year 3)
• Average time to bring a new team onboard and provision everything they need to deploy code (check in with teams to measure time to provision, target = baseline in Q1 2021, 5% improvement per Q, through 2021 then review and set next year’s targets)
• Number of teams migrating applications away from compliant public cloud platforms (count of teams migrating away)
• Net Promoter Score for Accelerated Public Cloud Services (by quarterly survey of all people actually working on one of our public cloud platforms, target = increase by 2 points per quarter in year 1, then review and set targets for next year) 

To address the last bullet in the Digital Annex Key Results, we wish to baseline as many of these as possible. First two are easy - could we do average time and Net Promoter Score?

**Which Sprint Goal is this issue related to?**
> Note: 'Milestone' is a ZenHub term that we use synonymously with 'Sprint'. 

The 'Milestone' description (created in ZenHub) should clearly list Sprint Goals. This section should indicate which 'goal' this issue is related to.

**Additional context**
- There are two sets of KPI efforts underway; neither should conflict with the other, in fact they will likely support each other.  We can proceed on both in parallel without any conflicts.  One is upward investment board / exec focused, the other is team managing their operational controls properly.

**Definition of done**
- Product Owner has confirmed that we'd like to use this KPI
- Describe our data collection strategy to measure Net Promoter Score (suggest survey via AirTable or Forms).
- Provide targets for progress against the indicators - suggest we measure our baseline then estimate how much we can improve per quarter (option to set "steady or improving" as a target)
- This will establish our baseline for the business case.
- Create another ticket for the next KPI that we'll baseline.

**Comment by scayzer** [comment]
Microsoft Forms survey fielded via product owners 2021-07-07
